### ISSUE TITLE
Getting SyntaxError: invalid syntax on `import tweepy`

### ISSUE THREAD

**Issue by guruprasaad123** [tweepy/tweepy#1344]
I installed [tweepy](https://github.com/tweepy/tweepy) for python on bash for windows using this command below.<br> 

```bash
pip3 install tweepy
```
And i was just getting started with tweepy authentication below.<br>
```python

import tweepy


auth = tweepy.OAuthHandler(consumer_key, consumer_secret)
auth.set_access_token(access_token, access_token_secret)

api = tweepy.API(auth)

print('authenticated - Twitter')
```

And i ran this command below to run this python3 script.
```bash
python3 twitter.py
```
As a result i got this Error  below . <br>please help me fix this.
```python
Traceback (most recent call last):
  File "twitter.py", line 19, in <module>
    import tweepy
  File "C:\Users\GURU\Anaconda3\envs\nlp\lib\site-packages\tweepy\__init__.py", line 17, in <module>
    from tweepy.streaming import Stream, StreamListener
  File "C:\Users\GURU\Anaconda3\envs\nlp\lib\site-packages\tweepy\streaming.py", line 355
    def _start(self, async):
                         ^
SyntaxError: invalid syntax
```

**Comment by guruprasaad123** [comment]
Installing the latest version as below fixed this error , thanks to this [post](https://github.com/tweepy/tweepy/issues/1017).
```bash
pip3 install -U git+https://github.com/tweepy/tweepy.git@2efe385fc69385b57733f747ee62e6be12a1338b
```

**Comment by Harmon758** [comment]
This is a duplicate of that issue, #1017, which was fixed with #1042 in Tweepy v3.7.0.
In the future, please make sure you're using the latest development version when reporting issues.
